client/dist/styles.css

```diff
@@ -0,0 +1,142 @@
+.likeBlock {
+    display: block;
+    width: 300px;
+    height: 96px;
+    /* border: 2px solid black; */
+    margin-bottom: 20px;
+}
+.header {
+    width: 300px;
+    height:31px;
+    border-bottom: 1px solid #f2f2f2
+
+}
+
+.likeHeader {
+    cursor: pointer;
+}
+
+.likeNumber{
+    line-height: 24px;
+
+    color: #999;
+    font-family: "Interstate","Lucida Grande","Lucida Sans Unicode","Lucida Sans",Garuda,Verdana,Tahoma,sans-serif;
+    font-size: 13px
+    
+}
+
+.viewAllLikes {
+    float: right;
+    line-height: 24px;
+    font-weight: 100;
+    color: #999;
+    font-family: "Interstate","Lucida Grande","Lucida Sans Unicode","Lucida Sans",Garuda,Verdana,Tahoma,sans-serif;
+    font-size: 13px
+}
+
+.viewAllLikes:hover {
+    color: black;
+}
+
+
+.likeContainer {
+    display: block;
+    width:300px;
+    height:65px;
+    /* padding-top: 31px; */
+}
+
+.profilePicture {
+    list-style: none;
+    padding-left: 9px;
+    padding-top: 5px;
+    /* padding-right: 10px; */
+    
+    width: 290px;
+    height: 40px;
+    cursor: pointer;
+}
+
+.pppics {
+    display: inline;
+
+    margin-left: -12px;
+    position: relative;
+    border: 2px solid white;
+
+}
+.roundImage {
+    position: relative;
+    height: 40px;
+    width: 40px;
+    border-radius: 50%;
+    border: 2px solid white;
+    /*account for border in pixel size to keep image true at 40px*/
+    box-sizing: border-box;
+
+    /* background-origin: border-box; */
+
+}
+
+.followHover {
+    
+    height: 90px;
+    width: 90px;
```

Comment:
Is this the line where you are trying to fix the width of the container? I haven't had a chance to look at the corresponding HTML, but one thing I might check: Is the followHover class on the pop-up container itself? Or the text?  If you are having issues styling the width, there might be something happening in the React-popups where you are not actually styling the container's width, but the content width.